charmhelpers/contrib/storage/linux/utils.py

```diff
@@ -123,6 +123,7 @@ def mkfs_xfs(device, force=False, inode_size=1024):
     cmd = ['mkfs.xfs']
     if force:
         cmd.append("-f")
-
-    cmd += ['-i', "size={}".format(inode_size), device]
+    if inode_size != 0:
```

Comment:
As suggested in the companion review you can just test for truthiness here `if inode_size:`